nessie/ops/nessie-bootstrap.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+apk add --no-cache util-linux
```

Comment:
That's for the 'script' that was missing, which brings us to the first comment (PTY etc.) above. If we ditch the usage of script, this will no longer be needed

Reply:
Agreed.  However if we do _not_ decide to get rid of `script` I still want to put this installation into the Nessie Dockerfile.  Otherwise I'm guessing _network access_ is required during the test -- making it flaky?

Reply:
Done - #2855